README.md

```diff
@@ -1,19 +1,24 @@
+![Kotlin](https://img.shields.io/badge/Kotlin-7f52ff?style=flat-square&logo=kotlin&logoColor=white) 
+![Kotlin Multiplatform](https://img.shields.io/badge/Kotlin%20Multiplatform-4c8d3f?style=flat-square&logo=kotlin&logoColor=white)
+![Compose Multiplatform](https://img.shields.io/badge/Jetpack%20Compose%20Multiplatform-000000?style=flat-square&logo=android&logoColor=white)
+
```

Comment:
[![Mobile-Wallet CI[Master/Dev]](https://github.com/openMF/mobile-wallet/actions/workflows/master_dev_ci.yml/badge.svg?branch=dev)](https://github.com/openMF/mobile-wallet/actions/workflows/master_dev_ci.yml) add this badge